_includes/header.html

```diff
@@ -1,46 +1,93 @@
-<header class="site-header">
-	<div class="branding">
-		{% if site.theme_settings.gravatar %}
-		<a href="{{ site.baseurl }}/">
-			<img class="avatar" src="https://secure.gravatar.com/avatar/{{ site.theme_settings.gravatar }}?s=100" alt="Bits & Bytes Home"/>
-		</a>
-		{% elsif site.theme_settings.avatar %}
-		<a href="{{ site.baseurl }}/">
-			<img class="avatar" src="{{ site.baseurl }}/{{ site.theme_settings.avatar }}" alt="Bits & Bytes Home"/>
-		</a>
-		{% endif %}
-		<a class="site-title" href="{{ site.baseurl }}/">{{ site.theme_settings.title }}
-			{% if site.theme_settings.subtitle %}
-				<br /><span class="site-subtitle">{{ site.theme_settings.subtitle }}</span>
-			{% endif %}
-		</a>
-	</div>
-	<nav class="site-nav">
-		<ul>
-			{% for page in site.pages %}
-			{% if page.title and page.hide != true %}
-			<li>
-				<a class="page-link" href="{{ page.url | relative_url }}">
-					{{ page.title }}
+<header class="header header--blue">
+	<div class="wrapper">
+	  <div class="container flex">
+		<div class="header__brand header__brand--text">
+		  <a href="/" class="header__brand-title">Bits & Bytes</a>
+		  <p class="header__brand-subtitle">
+			The Rockefeller Archive Center Blog
+		  </p>
+		</div>
+		<nav class="nav-right" aria-label="Main">
+			<ul class="nav__list show-on-lg-up">
+			  <li class="nav__item btn--navy">
+				<a
+				  class="nav__link"
+				  href="/authors"
+				  id="authors"
+				>
+				  Authors
+				  <span class="material-icon" aria-hidden="true">
+					arrow_right_alt
+				  </span>
+				</a>
+			  </li>
+			  <li class="nav__item btn--navy">
+				<a class="nav__link" href="/categories/" id="categories">
```

Comment:
From the Travis test fail message, it looks like this link should be `/categories`, not `/categories/`?